src/orchestrator/orchestrator.rs

```diff
@@ -813,11 +828,14 @@ impl<'a> JobTask<'a> {
                             )
                         })?;
                 }
-                self.bar.finish_with_message(format!(
-                    "[{} {} {}] Reusing artifact",
+                self.bar.set_message(format!(
+                    "{:-<MAX_ENDPOINT_NAME_LENGTH$} {:-<CONTAINER_ID_LENGTH$} {:-<JOB_UUID_LENGTH$} {} {} {} Reusing artifact",
+                    "", // endpoint_name
+                    "", // container_id
                     self.jobdef.job.uuid(),
+                    "██".white(),
```

Comment:
nit: White might not be ideal for terminals with white backgrounds but I'm not sure if there's a better alternative.